lib/sla.js

```diff
@@ -189,96 +243,276 @@ function originateChannel(client, inbound, bridge, trunkData) {
           unanswered.channel));
       hangup();
     });
+    updateState(client, bridge.name, 'NOT_INUSE');
     defer.reject(new customError.CustomError('InboundHungup',
           'Inbound channel hungup'));
   }
-
-  trunkData.getAllStations()
-    .then(function (stations) {
-      if (stations.length) {
-        stations.forEach(function(station) {
-          dialed.push({endpoint: station.endpoint, channel: client.Channel()});
-        });
-        dialed.forEach(function (object) {
-          var originate = Q.denodeify(object.channel.originate.bind(
-              object.channel));
-          originate({endpoint: object.endpoint, app: 'sla',
-              appArgs: 'dialed', timeout: 10})
-            .catch(function (err) {
-              defer.reject(err);
+  var defer = Q.defer();
```

Comment:
You could use vertical spacing throughout this pull request to make the code more readable. This section is especially dense.